scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -49,6 +49,24 @@ declare namespace WebdriverIO {
         };
     }
 
+    interface CucumberOptions {
```

Comment:
Please move this one to cucumber type definition file

Reply:
Moved to `cucumber-framework.d.ts`.
Will this throw error if user does not use `@wdio/cucumber-framework`?

Reply:
Shall I simply replace it with `object` like mochaOpts and jasmieOpts?